website/src/content/docs/index.mdx

```diff
@@ -15,7 +15,7 @@ hero:
   actions:
     - text: Get started
       icon: right-arrow
-      link: /guides/introduction/
```

Comment:
This broke the bottom links on the side when entering with `/` as it redirected to e.g. `/guides/introduction/api` instead of `/guides/api`.